requirements.yml

```
@@ -1,9 +1,15 @@
 ---
+collections:
+- name: containers.podman
+roles:
 - src: stackhpc.nfs
-# - src: stackhpc.openhpchttps://github.com/stackhpc/ansible-role-openhpc
+- src: https://github.com/stackhpc/ansible-role-openhpc
+  name: stackhpc.openhpc
+  version: feature/accounting
```

```suggestion
  version: master
```
Need to merge https://github.com/stackhpc/ansible-role-openhpc/pull/46 before we can revert this